Source/carthage/main.swift

```diff
@@ -9,8 +9,14 @@
 import CarthageKit
 import Commandant
 import Foundation
-import Result
+import ReactiveCocoa
 import ReactiveTask
+import Result
+
+guard ensureGitVersion().first()?.value == true else {
+	fputs("Carthage requires git \(CarthageRequiredGitVersion) or later. Please update your git version at first.\n", stderr)
```

Comment:
Fixed in 74ba78b. :zap:  `\n` is still needed as we already do [here](https://github.com/Carthage/Carthage/blob/ensure-git-version/Source/carthage/main.swift#L35).